pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1256,6 +1256,9 @@ type RemoteWriteSpec struct {
 	// MetadataConfig configures the sending of series metadata to the remote storage.
 	// +optional
 	MetadataConfig *MetadataConfig `json:"metadataConfig,omitempty"`
+
+	// Whether to enable HTTP2.
```

Comment:
```suggestion
	// Whether to enable HTTP2.
	// +optional
```

Reply:
:+1: 
I just added it and re-generated the doc. :heavy_check_mark: